### PR TITLE
Tweaks to some tests

### PR DESCRIPTION
**Pull request by rowanseymour**

### CODE REVIEW

File: temba/flows/models.py [hunk 1]
```diff
@@ -4922,16 +4922,16 @@ def as_json(self):
...
+                matches.append(value)
+            else:
+                return 0, None
 
         # all came out true, we are true
-        if sum == len(self.tests):
-            return 1, value
-        else:
-            return 0, None
+        return 1, " ".join(matches)
```
Comment (rowanseymour) [inline temba/flows/models.py]: My thinking here is that the AND test should return the matches of all of its tests combined.. right?

File: temba/flows/models.py [hunk 2]
```diff
@@ -5111,10 +5111,11 @@ def evaluate(self, run, sms, context, text):
 
         # see whether we start with our test
         if text.lower().find(test.lower()) == 0:
-            return 1, self.test
+            return 1, text[:len(self.test)]
```
Comment (rowanseymour) [inline temba/flows/models.py]: Ensures returned string has same case as input

File: temba/flows/models.py (hunk 1 above)
Comment (nicpottier) [inline temba/flows/models.py]: Ya, that seems more right for sure.

File: temba/flows/models.py [hunk 3]
```diff
@@ -5317,9 +5318,9 @@ class BetweenTest(NumericTest):
     MAX = 'max'
     TYPE = 'between'
 
-    def __init__(self, min, max):
-        self.min = min
-        self.max = max
+    def __init__(self, _min, _max):
+        self.min = _min
+        self.max = _max
```
Comment (nicpottier) [inline temba/flows/models.py]: Is this pep or something?

Comment (rowanseymour) [inline temba/flows/models.py]: yeah - min and max are builtin function names

**Comment by nicpottier** [comment]
:dart:

**Pull request merged**